.pre-commit-config.yaml

```diff
@@ -144,7 +144,7 @@ repos:
     # The `refs/tags/<tag>:refs/tags/<tag>` is needed for black's
     # `required-version` option to work.  See
     # https://github.com/psf/black/issues/2493#issuecomment-1081987650
-    rev: "refs/tags/22.3.0:refs/tags/22.3.0"
+    rev: "22.3.0"
```

Comment:
Don't remove `refs/tags/22.3.0:refs/tags/` in front of the version number, otherwise black will fail with exit code 1

    The required version `22` does not match the running version `0.1.dev1+gae2c075`!

because in pyproject.toml we set black's required-version option to 22.  See also the discussion https://github.com/psf/black/issues/2493#issuecomment-1081987650 and commit 0b35eafd3ee540fa1539d077edc30861db12d7b5.